.circleci/config/base.yml

```diff
@@ -645,7 +645,7 @@ step-nodejs-headers-build: &step-nodejs-headers-build
     name: Build Node.js headers
     command: |
       cd src
-      ninja -C out/Default third_party/electron_node:headers
```

Comment:
now that the prefix is `electron` and not `electron_node` we should probably change this to `electron:node_headers` to clarify what it is a bit more